source/manual/review-page.html.md

```diff
@@ -26,20 +24,14 @@ We also document Icinga alerts. These explain what to do when a certain alert tr
 - Update the page
```

Comment:
Could we add some explanation for how to do this for internal and external docs? I remember it being difficult the first time (having to find where the manual text was stored).